### PR TITLE
added prototype recursive tearing

### PR DESCRIPTION
**Pull request by vruge**
-cosmetic changes for simplifyLoops

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2015-06-30_23-16-08](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/264/console).

File: Compiler/BackEnd/Tearing.mo [hunk 1]
```diff
@@ -3740,5 +3748,310 @@ algorithm
   end matchcontinue;
 end findNEntries;
 
+// =============================================================================
+// section for preOptModule >>recursiveTearing<<
```
Comment (lochel) [inline Compiler/BackEnd/Tearing.mo]: This seems to be a post-optimization module.

**Comment by lochel** [comment]
Can you please describe briefly what the new optimization module is doing? Is it enabled per default? What are the different options that can be selected with the new flags?

File: Compiler/BackEnd/Tearing.mo [hunk 2]
```diff
@@ -85,6 +91,8 @@ end TearingMethod;
 public function tearingSystem "author: Frenkel TUD 2012-05"
   input BackendDAE.BackendDAE inDAE;
   output BackendDAE.BackendDAE outDAE;
+protected
+  Boolean update;
```
Comment (lochel) [inline Compiler/BackEnd/Tearing.mo]: This isn't used anywhere.

**Pull request closed without merging**